bundles/org.openhab.ui/web/src/components/item/item-form.vue

```diff
@@ -1,48 +1,81 @@
 <template>
   <div v-if="item" class="quick-link-form no-padding">
     <f7-list inline-labels no-hairlines-md>
-      <f7-list-input label="Name" type="text" placeholder="Required" :value="item.name"
-                     :disabled="!createMode" :info="(createMode) ? 'Note: cannot be changed after the creation' : ''"
-                     required :error-message="nameErrorMessage" :error-message-force="!!nameErrorMessage"
-                     @input="onNameInput" :clear-button="createMode" />
-      <f7-list-input label="Label" type="text" placeholder="Label" :value="item.label"
-                     @input="item.label = $event.target.value" :disabled="!editable" :clear-button="editable" />
-      <f7-list-item v-if="item.type && !hideType" title="Type" type="text" :disabled="!editable" smart-select :smart-select-params="{searchbar: true, openIn: 'popup', closeOnSelect: true}">
-        <select name="select-type" @change="item.type = $event.target.value">
-          <option v-for="t in types.ItemTypes" :key="t" :value="t" :selected="t === item.type.split(':')[0]">
+      <f7-list-group>
+        <f7-list-input label="Name" type="text" placeholder="Required" :value="item.name"
+                       :disabled="!createMode" :info="(createMode) ? 'Note: cannot be changed after the creation' : ''"
+                       required :error-message="nameErrorMessage" :error-message-force="!!nameErrorMessage"
+                       @input="onNameInput" :clear-button="createMode" />
+        <f7-list-input label="Label" type="text" placeholder="Label" :value="item.label"
+                       @input="item.label = $event.target.value" :disabled="!editable" :clear-button="editable" />
+      </f7-list-group>
+      <f7-list-group v-if="item.type && !hideType" :key="item.type">
+        <f7-list-input label="Type" type="select" :disabled="!editable" :value="item.type.split(':')[0]" @input="item.type = $event.target.value">
+          <option v-for="t in types.ItemTypes" :key="t">
             {{ t }}
           </option>
-        </select>
-      </f7-list-item>
-      <f7-list-item v-if="dimensions.length && item.type && !hideType && item.type.startsWith('Number')" title="Dimension" type="text" :disabled="!editable" smart-select :smart-select-params="{searchbar: true, openIn: 'popup', closeOnSelect: true}">
-        <select name="select-dimension" @change="setDimension($event.target.value)">
-          <option key="Number" value="Number" :selected="item.type === 'Number'" />
-          <option v-for="(d, i) in dimensions" :key="d.name" :value="i" :selected="'Number:' + d.name === item.type">
+        </f7-list-input>
+
+        <!-- Dimensions -->
+        <f7-list-input v-if="dimensions.length && item.type.startsWith('Number')"
```

Comment:
I must admit, I don't like the behaviour of the built in f7 select/dropdowns used in this PR and I too would prefer the search input like in the popup. However, I don't like it being a "popup". Popups make the entire process more tedious and involves more clicks than necessary, or at least it feels that way to me.

I'll look around for a better solution, perhaps using an additional component that offers enhanced dropdown features.

Reply:
In case of the Item type and the dimension, it's no additional click compared to the dropdown. 
One click to open the popup, and another click to select the type.
That's the same number of clicks as with the dropdown.

Using a popover might also be an option, but only for the type, but it seems it is not searchable:
![image](https://github.com/openhab/openhab-webui/assets/73423173/94dc670c-0689-4cc0-b95a-d8b4ad90a2b5)

(https://framework7.io/docs/smart-select)